sdbuild/packages/xrt/pre.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Apparently this file is causing failure on ZCU104 build. It seems to have wrong permissions (missing +x)

Reply:
Got it. Will change.